.github/workflows/release.yml

```diff
@@ -0,0 +1,142 @@
+name: "Release New Version"
+run-name: "Release ${{ inputs.version }}"
+
+on:
+  workflow_dispatch:
```

Comment:
`workflow_dispatch` allows a manual trigger by anyone with write access to the repository. If we want to limit the number of people that can start a release, we'd have to add checks in this workflow, but I don't think that's necessary at this point.